modules/ROOT/pages/setup-edit.adoc

```diff
@@ -0,0 +1,55 @@
+= Set Up Partner Manager
+
+Before you can deploy message flows from Partner Manager, you must set up and configure Anypoint Partner Manager for use by the team members in your organization who configure, deploy, and monitor B2B Trading Partner profiles and message flows.
+
+== Prerequisites
+
+Before you set up Partner Manager, you must have:
+
+* An Aypoint Platform user account with the Anypoint Partner Manager entitlement.
+* The Partner Manager Administrator role, which assigns permission levels for all users who need access to B2B message flows in their environment (Production or  Sandbox).
+* The required Mule vCore capacity to handle your B2B transaction workload needs. 
+* Licenses for the connectors that are used in your B2B message flows based on the protocols and message formats you are using:
+* AS2 connector
+* X12 connector
+* Anypoint Studio 7.4 or later in order to create DataWeave maps for EDI data transformation.
+
+If you require assistance, contact your account representative.
+
+== Storage Requirements
+
+Storage for the B2B transmission payloads you send or receive, such as a database, Amazon S3, Azure Blob Storage, and so on. +
+For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Payload Storage API for Partner Manager].
+
+== Deployment Requirements
+
+* Mule runtime engine (Mule) 4.2.2 or later (for on-premises only).
+* Keep the default deployment of message flows to CloudHub (for information on CloudHub settings, refer to xref:cloudhub-deploy-options.adoc[CloudHub Deployment Options], or deploy to your own hosted on-premise Mule runtime, by clicking *Stop deploying to CloudHub* and going to  xref:deploy-onpremise.adoc[Deploy On-Premises for Partner Manager]).
+
+== Parter Manager Permissions
+
+You can assign the following permissions, per Anypoint Platform environment, to your Partner Manager users:
+
+*  *Partner Manager Administrator* +
+Has full permissions.
```

Comment:
```suggestion
Has full permissions
```
This is kind of awk, having just one characteristic when the others have two or more and they are bulleted. But I don't see a solution.